### ISSUE TITLE
Bayesian Linear Regression Tutorial

### ISSUE THREAD

**Issue by markaduol** [tensorflow/probability#168]
I'm working on a tutorial for Bayesian Linear Regression as my first contribution. Do you think that this is something that could be useful here?

**Comment by henrysky** [comment]
Yes I think that will be useful as I just opened an issue #166 , I cannot even get `DenseFlipout` and `DenseReparameterization` to do a simple y=mx+c regression task and get reasonable uncertainty.

**Comment by axch** [comment]
We'd be very happy to have a contribution along those lines!  Just to make sure you know the context, we have some general linear modeling facilities in TFP: https://www.tensorflow.org/probability/api_docs/python/tfp/glm; and I assume you are aiming for something a little more introductory than https://github.com/tensorflow/probability/blob/master/tensorflow_probability/examples/jupyter_notebooks/HLM_TFP_R_Stan.ipynb.
